release-notes/opensearch-security.release-notes-2.14.0.0.md

```diff
@@ -0,0 +1,40 @@
+## Version 2.14.0.0
+
+Compatible with OpenSearch 2.14.0
+
+### Enhancements
+* Check for and perform upgrades on security configurations ([#4251](https://github.com/opensearch-project/security/pull/4251))
+* Replace bouncy castle blake2b ([#4284](https://github.com/opensearch-project/security/pull/4284))
+* Adds saml auth header to differentiate saml requests and prevents auto login as anonymous user when basic authentication fails ([#4228](https://github.com/opensearch-project/security/pull/4228))
+* Dynamic sign in options ([#4137](https://github.com/opensearch-project/security/pull/4137))
+* Add index permissions for query insights exporters ([#4231](https://github.com/opensearch-project/security/pull/4231))
+* Add new stop words system index ([#4181](https://github.com/opensearch-project/security/pull/4181))
+* Add query assistant role and new ml system indices ([#4142](https://github.com/opensearch-project/security/pull/4142))
```

Comment:
This was also included in 2.13: https://github.com/opensearch-project/security/pull/4143